docs/docs/index.md

```diff
@@ -155,16 +155,44 @@ run/debug configuration.
 
 ### VSCode
 
-Add following in the `settings.json`:
+Add the following to `.vscode/settings.json`:
 
 ```json
 {
-  ...
+  "python.pythonPath": "/usr/bin/python",
   "python.autoComplete.extraPaths": ["__pypackages__/<major.minor>/lib"],
   "python.analysis.extraPaths": ["__pypackages__/<major.minor>/lib"]
 }
 ```
 
+Change the Python interpreter path to the one you want to use for the current project.
+
+To allow debugging your project in VSCode, set the `PYTHONPATH` environment variable in your launch configuration, in `.vscode/launch.json`.
+For example, to debug your `pytest` run:
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "pytest",
+            "type": "python",
+            "request": "launch",
+            "module": "pytest",
+            "args": ["tests"],
+            "justMyCode": false,
+            "env": {"PYTHONPATH": "__pypackages__/3.9/lib"}
+        }
+    ]
+}
+```
+
+If your package resides in a `src` directory, add it to `PYTHONPATH` as well:
+
+```json
+            "env": {"PYTHONPATH": "src:__pypackages__/3.9/lib"}
```

Comment:
I think we don't need this given the current project is installed into lib

Reply:
Oh, you might be right, I'll run some more tests to clarify this.

Reply:
When PEP582 is not enabled, it does not work without `src` in `PYTHONPATH`.

Reply:
I've opted for a collapsible note to document this.